src/backend/db/queries.js

```diff
@@ -1,389 +1,382 @@
-const knex = require('./knex.js');
+const knex = require("./knex.js");
 
 //Users
 
-const addUser = async ({name, pwd}) => {
-  return knex('Users')
-         .insert({name: name, pwd: pwd})
-         .returning('name')
-}
+const addUser = async ({ name, pwd }) => {
```

Comment:
knex's query builder is safe to use by default so this is fine over raw sql but its worth learning how to write raw sql yourself.

also table names are case insensitive so no need to capitalize the first letter